### DISCUSSION TITLE
Plugin Store - Improve PNG quality of screenshots

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/console#88] [Plugin Store]
Take for example the [Hyper](https://plugins.craftcms.com/hyper?craft5) plugin, which has high-res retina-ready images. These are noticeably blurry due to the image transform used, and it's pretty common with Craft installs from what I've found when it comes to PNG images when resized. Tweaking Imagick might be an option.

Note the [transformed](https://pluginscreenshots.craft-cdn.com/hyper/_2200x2200_fit_center-center_none/2-hyper-field-settings.png?1695120356) image and the [non-transformed image](https://pluginscreenshots.craft-cdn.com/hyper/2-hyper-field-settings.png?1695120356), which visually is below (transformed on the left).

<img width="1197" alt="image" src="https://github.com/craftcms/console/assets/1221575/7ac5696b-ad6e-4ce8-94a6-c49d9b28c926">

Text is noticeably blurry zoomed up, but is also noticeable at just the maximised image level.

I'm aware this is low-priority in the scheme of things!

**Comment by timkelty** [comment]
Possibly related: https://github.com/craftcms/cms/issues/10371
